File: 02 - Data Science/06 - CursoPOOUber/Python/UberVan.py
from car import Car

class UberVan(Car):
    typeCarAccepted = []
    seatsMaterial = []

    def __init__(self, License, Driver, typeCarAccepted, seatMaterial):
        super().__init__(License, Driver)
        self.typeCarAccepted = typeCarAccepted
        self.seatsMaterial = seatMaterial

File: 02 - Data Science/06 - CursoPOOUber/PHP/UberVan.php
<?php
require_once('Car.php');
class UberVan extends Car {
    public $typeCarAccepted;
    public $seatsMaterials;

        public function __construct($license, $driver, $typeCarAccepted, $seatsMaterials){
            parent::__construct($license, $driver);
            $this->typeCarAccepted = $typeCarAccepted;
            $this->seatsMaterials  = $seatsMaterials;
        }

        public function setPassenger($passenger){
            if ($passenger == 6){
                $this->passenger = $passenger;        
            }
            else{
                echo "Necesitas asignar 6 pasajeros"; 
            }
        }


        
}
?>